preview-src/image-borders.adoc

```diff
@@ -0,0 +1,75 @@
+= Image display options
+:page-theme: docs
+// :page-styles: image-border
+
+
+[abstract]
+--
+Display images with border or box-shadow
+--
+
+- set `:page-styles: image-border` or `:page-styles: image-shadow` in a playbook, _antora.yml_ or in a page header to add border or shadow to all pages in the docset, component, or page.
+- set `[.border]` or `[.shadow]` for individual images
+- combine border and shadow settings to apply both styles
+- use `[.no-border]` or `[.no-shadow]` to override a global setting for a specific image
+
+
+== Border
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```

Comment:
Do we need the latin fluff? I'm all for it if we do!

Reply:
The lorem ipsum is just to illustrate how images look with some surrounding text.